components/PaywallModal.vue

```diff
@@ -0,0 +1,321 @@
+<template>
+  <UModal
+    :full-screen="props.isFullScreen || isMobileScreen"
+    :dismissible="props.isDismissible"
+    :ui="{ content: 'phone:rounded-2xl overflow-x-hidden phone:w-[90vw] max-w-[840px]' }"
+  >
+    <template #content>
+      <div class="flex flex-col phone:flex-row w-full h-full">
+        <div class="w-full relative bg-[#fbfbfb]">
+          <UButton
+            v-if="props.isDismissible"
+            icon="i-material-symbols-close"
+            class="
+              phone:hidden absolute top-4 right-4 cursor-pointer
+            "
+            variant="link"
+            size="md"
+            @click="handleCloseButtonClick"
+          />
+          <img
+            :src="paywallHeaderImg"
+            alt="Paywall Header"
+            class="phone:absolute top-0 left-0 w-full object-cover"
+          >
+          <img
+            :src="paywallFooterLogo"
+            alt="Paywall Footer"
+            class="hidden phone:block absolute bottom-0 left-0 w-full object-cover"
+          >
+
+          <div class="absolute bottom-0 phone:inset-0 flex flex-col items-center justify-center px-10 w-full">
+            <div class="phone:hidden bg-black rounded-full px-6 py-2">
+              <span class="font-bold text-white">{{ $t('pricing_page_subscription') }}</span>
+            </div>
+            <img
+              :src="paywallBodyLogo"
+              alt="3ook Logo"
+              class="w-full max-w-[300px] phone:max-h-[200px] object-contain"
+            >
+          </div>
+        </div>
+
+        <div class="flex flex-col justify-center items-start p-5 phone:p-12 w-full">
+          <div class="flex flex-col items-start mx-6 gap-2">
+            <div class="hidden phone:block bg-black rounded-full px-6 py-2 mb-3">
+              <span class="font-bold text-white">{{ $t('pricing_page_subscription') }}</span>
+            </div>
```

Comment:
<img width="370" alt="image" src="https://github.com/user-attachments/assets/a38ac512-0a31-40be-a7e0-4e2da6b5cb0e" />

```suggestion
            <div
              class="hidden phone:block mb-3 px-6 py-2 text-white text-center font-bold bg-black rounded-full"
              v-text="$t('pricing_page_subscription')"
            />
```